lib/api/1.1/northbound/nodes.js

```diff
@@ -540,12 +546,14 @@ function nodesRouterFactory (
     router.get('/nodes/:identifier/workflows/active', rest(function (req) {
         return waterline.nodes.needByIdentifier(req.params.identifier)
         .then(function (node) {
-            return taskGraphProtocol.getActiveTaskGraph( { target: node.id })
-            .then(function (graph) {
-                if (graph) {
-                    return waterline.graphobjects.findOne({ instanceId: graph.instanceId });
-                }
-            });
+            return workflowApiService.findActiveGraphForTarget(node.id);
```

Comment:
Nothing wrong here, but begged the question - if you leverage the new entrypoint to start a workflow that doesn't have a target set to a `node`/`node.id`, how do you query for it's state or otherwise lookup status like you do here for `/nodes/:id/workflows/active`?